cmd/mq_prometheus/exporter.go

```diff
@@ -189,32 +275,44 @@ func newMqGaugeVec(elem *mqmetric.MonElement) *prometheus.GaugeVec {
 		labels,
 	)
 
-	log.Infof("Created gauge for %s", elem.MetricName)
+	log.Infof("Created gauge for %s%s", prefix, elem.MetricName)
```

Comment:
Will prefix always end in a / ? or will elem.MetricName always start with one?

Just thinking if they don't there could be times where you can't tell difference between prefix and elem.MetricName.

Reply:
That line is just reporting the gauge name for debug purposes. I found it better to  print it out that way. The prefix always ends with "_" - it's coded in the same function